chessence-backend/backend_with_db.js

```diff
@@ -15,7 +15,7 @@ var corsOptions = {
     origin: process.env.CORS_ORIGIN,
     credentials: true,
 };
-app.use(cors());
+app.use(cors(corsOptions));
```

Comment:
not sure if @LMWTT1230  is done testing azure, he removed corsOptions from `app.use` but re-adding it should be fine?

Reply:
Yeah, I noticed that. I needed to add it back for testing this locally. If need be, we will update it.